pkg/sidecar/sidecar.go

```diff
@@ -411,3 +428,16 @@ func (w JWTBundlesWatcher) OnJWTBundlesWatchError(err error) {
 		w.sidecar.config.Log.Errorf("Error while watching JWT bundles: %v", err)
 	}
 }
+
+func (s *Sidecar) CheckHealth() bool {
```

Comment:
Should this all check the x509WriteSuccess?

Reply:
Oops. Yes.